src/test/java/com/wikia/webdriver/testsuites/testSuite.xml

```diff
@@ -107,7 +112,9 @@
             <class name="com.wikia.webdriver.testcases.discussions.UploadingImageTests"/>
             <class name="com.wikia.webdriver.testcases.discussions.UpvotingTests"/>
             <class name="com.wikia.webdriver.testcases.discussions.ZeroErrorStateTests"/>
-            <class name="com.wikia.webdriver.testcases.auth.FacebookTests"/>
+            <class name="com.wikia.webdriver.testcases.fandom.HeroUnitTest"/>
```

Comment:
Could you check if this HeroUnitTests are still valid? AFAIR this feature was removed from app loooong time ago and probably these tests are not run in any job. It may be a separate ticket

Reply:
this is Fandom related so I don't have much expertise on it